lib/sequent/core/helpers/attribute_support.rb

```diff
@@ -60,6 +62,7 @@ def attrs(args)
             class_eval <<EOS
               def update_all_attributes(attrs)
                 super if defined?(super)
+                ensure_known_attributes(attrs)
```

Comment:
This may only throw an error on instatiation, not deserialization. Otherwise existing events can not be deserialized.